packages/replayio/src/utils/initialization/promptForNpmUpdate.ts

```diff
@@ -1,29 +1,50 @@
-import { name } from "../../../package.json";
+import { name as packageName } from "../../../package.json";
 import { prompt } from "../prompt/prompt";
 import { updateCachedPromptData } from "../prompt/updateCachedPromptData";
 import { highlight } from "../theme";
+import { getPackageManagerCommand } from "./getPackageManagerCommand";
 import { UpdateCheckResult } from "./types";
 
 const PROMPT_ID = "npm-update";
 
 export async function promptForNpmUpdate(updateCheck: UpdateCheckResult<string>) {
   const { fromVersion, toVersion } = updateCheck;
 
-  console.log("");
-  console.log("A new version of replayio is available!");
-  console.log("  Installed version:", highlight(fromVersion));
-  console.log("  New version:", highlight(toVersion));
-  console.log("");
-  console.log("To upgrade, run the following:");
-  console.log(highlight(`  npm install -g ${name}`));
-  console.log("");
-  console.log("Press any key to continue");
-  console.log("");
+  const command = getPackageManagerCommand();
+  if (command) {
```

Comment:
>  I think everyone who uses a package manager other than NPM knows how to convert the standard "npm install" message to their own package-manager-of-choice.

Right. So I wasn't worried about what we print here - just that we don't execute the command when the user confirms the prompt. We execute runtime installs - that's of course much more helpful since the user doesn't have to know how to do that from the top of their head. It's just that when we are prompting for both it feels off that they both don't execute things. 

That was my literal confusion as the user here. I was first prompted about the runtime update and later about the npm update (during some subsequent run). At that point, I was just expecting that it would also perform the installation for me but it didn't.

Reply:
I think that without this auto-installation it might not be worth pulling in this PR. I understand that auto-installation isn't exactly a no-brainer with a plethora of alternative package managers on the market, use cases and setups. So staying away from it might very well be just the right thing to do to avoid edge cases and weird problems.

Reply:
Having looked into this, and thought more about the edge cases, I'm inclined to agree. I posted this PR for discussion but I'm happy to bail on it and just do [this one thing](https://github.com/replayio/replay-cli/pull/374#discussion_r1562227248) instead.